core/src/apps/cardano/sign_tx.py

```diff
@@ -569,3 +582,14 @@ def _should_hide_output(output: List[int], inputs: List[CardanoTxInputType]) ->
         ):
             return False
     return True
+
+
+def _is_network_id_verifiable(
+    outputsCount: int, withdrawalsCount: int, hasStakePoolRegistration: bool
```

Comment:
please use snake_case for variable names

also return type annotation is missing